content/en/docs/tasks/administer-cluster/topology-manager.md

```diff
@@ -24,7 +24,7 @@ _Topology Manager_ is a Kubelet component that aims to co-ordinate the set of co
  
 
 
-## {{% heading "prerequisites" %}}
+## {{% heading "prerequisites" %}}

```

Comment:
Hi @k-wiatrzyk ,
Is there a change here?

Reply:
This must be a newline change. I will fix it.

Reply:
@kbhawkey 
It was an editor fault, it changed this by himself. Fixed.

Reply:
👍